Framework/Kernel/src/InstrumentInfo.cpp

```diff
@@ -284,7 +285,7 @@ void InstrumentInfo::fillLiveData(const Poco::XML::Element *elem) {
   for (unsigned long i = 0; i < connections->length(); ++i) {
     auto *conn = dynamic_cast<Poco::XML::Element *>(connections->item(i));
     try {
-      m_listeners.emplace_back(LiveListenerInfo(conn));
+      m_listeners.emplace_back(LiveListenerInfo(this, conn));
```

Comment:
As discussed, this calls the move constructor rather than the copy constructor. But you're still right, even the move constructor call can be avoided by just passing in the construction arguments.